packages/itwin/grouping-mapping-widget/src/widget/components/GroupPropertyUtils.ts

```diff
@@ -0,0 +1,360 @@
+import type { IModelConnection } from "@itwin/core-frontend";
+import type { ECProperty } from "@itwin/insights-client";
+import { DataType } from "@itwin/insights-client";
+import type {
+  ContentDescriptorRequestOptions,
+  Field,
+  KeySet,
+  NestedContentField,
+  PropertiesField,
+  Ruleset,
+  RulesetVariable,
+  StructFieldMemberDescription,
+} from "@itwin/presentation-common";
+import {
+  ContentSpecificationTypes,
+  DefaultContentDisplayTypes,
+  PropertyValueFormat,
+  RelationshipMeaning,
+  RuleTypes,
+} from "@itwin/presentation-common";
+import { Presentation } from "@itwin/presentation-frontend";
+import { deepEqual } from "fast-equals";
+
+export interface PropertyMetaData {
+  // Display label
+  label: string;
```

Comment:
Instead of adding comments, you could just rename the properties:
```
label -> displayLabel
schema -> sourceSchema
className -> sourceClassName
type -> propertyType
```
etc.

Reply:
Changed them. Let me know what you think.

Reply:
Looks better now.